src/test/resources/backendResponses/informationResponse.json

```diff
@@ -50,6 +50,6 @@
         "ttlValue" : 600
     },
     "erUrl" : {
-        "url" : "http://localhost:8080/search/"
+        "url" : "http://localhost:8080/eventrepository/search/"
```

Comment:
Isnt this a mocked fake response and thus could be anything? The er port is not 8080 as well?

Reply:
Yes, it is mocked. But I wanted to be consistent.  And ER port is set to 8080 as default in property file.